bin/deploy-vhost

```diff
@@ -236,7 +238,11 @@ sub nginx_graceful {
 
 sub graceful {
     # Prompt webserver to reread files, on machines which have apache...
-    apache_graceful();
+    if (!$conf->{touch_to_restart} || $restart_apache) {
```

Comment:
If `touch_to_restart` isn't present won't this short circuit the check on`$restart_apache` and run `apache_graceful()` every time? Shouldn't it be `&&`, or something like:
```
if ( !$conf->{touch_to_restart} {
  apache_graceful() if $restart_apache;
}
```